Content.Server/GameObjects/EntitySystems/AntimatterEngineSystem.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Content.Server.GameObjects.Components.NodeContainer.NodeGroups;
+using Content.Server.GameObjects.Components.Power.AME;
+using JetBrains.Annotations;
+using Robust.Shared.GameObjects.Systems;
+using Robust.Shared.Interfaces.Configuration;
+using Robust.Shared.IoC;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Content.Server.GameObjects.EntitySystems
+{
+    [UsedImplicitly]
+    public class AntimatterEngineSystem : EntitySystem
+    {
+        [Dependency] private readonly IConfigurationManager _config = default!;
+        private int _injectionTickCounter = 0;
+
+        public override void Initialize()
+        {
+            base.Initialize();
+        }
+
+        public override void Update(float frameTime)
+        {
+            base.Update(frameTime);
+            _injectionTickCounter++;
+            if (_injectionTickCounter >= 5 * (_config.GetCVar<int>("net.tickrate")))
```

Comment:
This can be changed to instead store an increment of `frameTime` and checking when that is equal to or bigger than 5, like StomachBehavior does. After that's changed, the counter should be decreased by the threshold instead of reset to 0 (https://github.com/space-wizards/space-station-14/pull/1617)

https://github.com/space-wizards/space-station-14/blob/ed1a96e536026046c4ad6c2c73f7968f09e1205c/Content.Server/Body/Mechanisms/Behaviors/StomachBehavior.cs#L27-L33